docker/compose/docker-compose.yml

```diff
@@ -79,8 +79,11 @@ services:
     ports:
       - "12345:12345"
     command: segmentstore
+    privileged: true
     environment:
       WAIT_FOR: bookie1:3181,bookie2:3182,bookie3:3183,hdfs:8020
```

Comment:
Does Pravega work with a single bookkeeper? If it does, we could could have a single bookie, then scale it with something like `docker-compose scale bookie=5`.  `docker-compose` will always start with a single instance of each service though, so this only really works if we can run with a single bookkeeper.

The docker compose file wasn't intended to be particularly flexible. We just wanted an easy way to spin up a full Pravega. The docker swarm stuff is a better choice if you need to scale.